package.json

```diff
@@ -28,7 +28,8 @@
     "./CHANGELOG.md"
   ],
   "scripts": {
-    "build": "npm run build:clean && npm run i18n:generate && vite build",
+    "build": "npm run build:clean && npm run i18n:generate && tsc && tsc-alias && npm run build:styles",
+    "build:styles": "postcss \"src/**/*.module.scss\" --base src --dir dist --verbose",
```

Comment:
Why not use `sass-embedded`? It's already in the package, so it would remove the need for postcss here